data/repository/build.gradle.kts

```diff
@@ -48,35 +48,43 @@ kotlin {
                 implementation(projects.domain.authentication)
                 implementation(libs.io.insert.koin.core)
                 implementation(libs.kotlinx.coroutines.core)
+                api(libs.store)
+                implementation ("com.squareup.sqldelight:coroutines-extensions:1.5.4")
```

Comment:
please update version catalog and reference library type-safe

Reply:
will do before landing! thanks for catching